egs/wsj/asr2/run.sh

```diff
@@ -0,0 +1,370 @@
+
+    ${cuda_cmd} --gpu ${ngpu} ${expdir}/train.log \
+        asr_train.py \
+        --accum-grad ${accum_grad} \
+        --ngpu ${ngpu} \
+        --backend ${backend} \
+        --outdir ${expdir}/results \
+        --debugmode ${debugmode} \
+        --dict ${dict} \
+        --debugdir ${expdir} \
+        --minibatches ${N} \
+        --verbose ${verbose} \
+        --resume ${resume} \
+        --seed ${seed} \
+        --train-json ${feat_tr_dir}/data.json \
+        --valid-json ${feat_dt_dir}/data.json \
+        --elayers ${elayers} \
+        --eunits ${eunits} \
+        --subsample ${subsample} \
+        --dlayers ${dlayers} \
+        --dunits ${dunits} \
+        --aheads ${aheads} \
+        --adim ${adim} \
+        --mtlalpha ${mtlalpha} \
+        --batch-size ${batchsize} \
+        --dropout-rate ${dropout} \
+        --maxlen-in ${maxlen_in} \
+        --maxlen-out ${maxlen_out} \
+        --opt ${opt} \
+        --grad-clip ${grad_clip} \
+        --sampling-probability ${samp_prob} \
+        --epochs ${epochs} \
+        --sortagrad ${sortagrad} \
+        --lsm-weight ${lsm_weight} \
+        --model-module "espnet.nets.${backend}_backend.asr_transformer" \
+        --transformer-lr ${lr_init} \
+        --transformer-warmup-steps ${warmup_steps} \
+        --transformer-input-layer ${input_layer} \
+        --transformer-attn-dropout-rate ${attn_dropout} \
+        --transformer-length-normalized-loss ${len_norm} \
+        --transformer-init ${ninit} \
+        --patience ${patience}
+fi
+
+if [ ${stage} -le 5 ] && [ ${stop_stage} -ge 5 ]; then
+    echo "stage 5: Decoding"
+    nj=32
+
+    pids=() # initialize pids
+    for rtask in ${recog_set}; do
+    (
+        decode_dir=decode_${rtask}_beam${beam_size}_e${recog_model}_p${penalty}_len${minlenratio}-${maxlenratio}_ctcw${ctc_weight}_rnnlm${lm_weight}_${lmtag}
+        if [ ${use_wordlm} = true ]; then
+            recog_opts="--word-rnnlm ${lmexpdir}/rnnlm.model.best"
+        else
+            recog_opts="--rnnlm ${lmexpdir}/rnnlm.model.best"
+        fi
+        if [ ${lm_weight} == 0 ]; then
+            recog_opts=""
+        fi
+        feat_recog_dir=${dumpdir}/${rtask}/delta${do_delta}
+
+        # split data
+        splitjson.py --parts ${nj} ${feat_recog_dir}/data.json
+
+        #### use CPU for decoding
+        ngpu=0
+
+        ${decode_cmd} JOB=1:${nj} ${expdir}/${decode_dir}/log/decode.JOB.log \
+            asr_recog.py \
+            --ngpu ${ngpu} \
+            --backend ${backend} \
+            --recog-json ${feat_recog_dir}/split${nj}utt/data.JOB.json \
+            --result-label ${expdir}/${decode_dir}/data.JOB.json \
+            --model ${expdir}/results/${recog_model}  \
+            --beam-size ${beam_size} \
+            --penalty ${penalty} \
+            --maxlenratio ${maxlenratio} \
+            --minlenratio ${minlenratio} \
+            --ctc-weight ${ctc_weight} \
+            --lm-weight ${lm_weight} \
```

Comment:
We can keep this, but please add some WARNING in the python program that this is not implemented yet.